src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -118,13 +118,18 @@ class TypeLookupManager
 
     /**
      * Constructor
-     * @param prot Pointer to the BuiltinProtocols object.
      */
-    TypeLookupManager(
-            fastrtps::rtps::BuiltinProtocols* protocols);
+    TypeLookupManager();
 
     virtual ~TypeLookupManager();
 
+    /**
+     * Initialize the RTPSParticipantImpl BuiltinProtocols and temp ReaderProxyData and WriterProxyData
+     * of the TypeLookupManager
+     * @param prot Pointer to the BuiltinProtocols object.
+     */
```

Comment:
I am missing the possible return values and in which cases either one is returned.